Dockerfile

```diff
@@ -19,6 +19,10 @@ ADD . /src/github.com/openwhisk/openwhisk-cli
 RUN echo "Restoring Go dependencies"
 RUN cd /src/github.com/openwhisk/openwhisk-cli && /bin/godep restore -v
 
+# Collect all translated strings into single .go module
+RUN echo "Packaging i18n Go module"
+RUN cd /src/github.com/openwhisk/openwhisk-cli && /bin/go-bindata -pkg wski18n -o wski18n/i18n_resources.go wski18n/resources
```

Comment:
I do not think we need them any more, because i18n module is accessible in the source code. No need to build them .